src/planner/expand_hypertable.c

```diff
@@ -327,10 +337,53 @@ constify_timestamptz_op_interval(PlannerInfo *root, OpExpr *constraint)
 									constraint->inputcollid);
 }
 
+static Datum
+calculate_timestamptz_timezone_bucket(int strategy, Datum width, Datum value, List *args)
```

Comment:
I don't see 2nd argument "width"  being used anywhere in this function. Is it required ?
There is a C flag which when enabled will cause build failures if a function argument is never used inside the function.

Reply:
Ohhh... it was a leftover! What is the CFLAG??

Reply:
gcc -v
Apple clang version 14.0.0 (clang-1400.0.29.202)
Target: arm64-apple-darwin22.1.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin

gcc --help | grep -i 'unused-arguments'
  -Qunused-arguments      Don't emit warning for unused driver arguments

I think gcc on unix platforms has a flag called -W-unsed-parameter.